package.json

```diff
@@ -1566,6 +1586,15 @@
                     }
                 }
             },
+            {
+                "command": "aws.cdk.renderStateMachineGraph",
+                "title": "%AWS.command.cdk.renderStateMachineGraph%",
```

Comment:
There are duplicate command entries with different titles. I'm guessing the idea was to have different text for when the user right clicks on a node versus when they use the command palette. VS Code does not have this context, so it just takes the 'previewStateMachine' title. So you'll probably need to split these into two separate commands and register both: one that takes a `ConstructNode` as an argument, and one that takes no arguments. 

Or just delete the first entry. The right-click text on the construct node will then be verbose but clear in what it does.

Reply:
Another good catch, removed the duplicate entry in favour of having the more verbose wording. Justin had previously requested to consolidate the two commands into a single one, so happy to leave that as is.